Base/Dockerfile

```diff
@@ -61,6 +61,8 @@ ENV HOME=/home/seluser
 # Add Grid check script
 #======================================
 COPY check-grid.sh entry_point.sh /opt/bin/
+# change right for /opt/bin*.sh file
+RUN chmod 775 /opt/bin/*.sh
```

Comment:
Please do the `COPY` with `chmod` as mentioned in the Docker documentation https://docs.docker.com/engine/reference/builder/#copy